asserts/asserts.go

```diff
@@ -0,0 +1,285 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2014-2015 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package asserts
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"unicode/utf8"
+)
+
+// AssertionType labels assertions of a given type
+type AssertionType string
+
+// Understood assertions
+const (
+// IdentityType AssertionType = ...
+// AccountKeyType AssertionType = "account-key"
+// ...
+)
+
+// Assertion signature format types
+const (
+	OpenPGPSig = "openpgp"
```

Comment:
Sorry, I now realize my comment was far from clear. I was suggesting dropping the exported constant, specifically, while it's the only possible constant and irrelevant for consumers of that API. The signature prefix looks fine.